src/main/java/org/traccar/protocol/NavtelecomProtocolDecoder.java

```diff
@@ -269,7 +269,7 @@ protected Object decode(
                                     position.set("freq" + (j + 2 - 35), buf.readUnsignedShortLE());
                                     break;
                                 case 37:
-                                    position.set(Position.KEY_HOURS, buf.readUnsignedIntLE());
+                                    position.set(Position.KEY_HOURS, buf.readUnsignedIntLE() / 60);
```

Comment:
Also this is not correct. Hours parameter is stored in milliseconds.

Reply:
got it, i'll fix it